### PR TITLE
Subscription For the Web version using PayPal 

### PR DESCRIPTION
**Pull request by martinbedouret**

### CODE REVIEW

File: src/components/Settings/Settings.component.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ export class Settings extends PureComponent {
       }
     ];
 
-    if (isAndroid() && !isInFreeCountry) {
+    if (!isInFreeCountry) {
```
Comment (RodriSanchez1) [inline src/components/Settings/Settings.component.js]: Add `!isIos && !isElectron`

File: src/components/Settings/Subscribe/Subscribe.container.js [hunk 2]
```diff
@@ -47,11 +47,12 @@ export class SubscribeContainer extends PureComponent {
 
   handleRefreshSubscription = () => {
     const { updateIsSubscribed, updatePlans } = this.props;
-    //window.CdvPurchase.store.restorePurchases();
     updateIsSubscribed();
     updatePlans();
   };
 
+  handleCancelSubscription = () => {};
```
Comment (RodriSanchez1) [inline src/components/Settings/Subscribe/Subscribe.container.js]: Delete this function, i think its defined below

File: src/components/Settings/Subscribe/SubscriptionInfo.js [hunk 3]
```diff
@@ -114,11 +118,18 @@ const subscriptionInfo = ({
...
-            window.CdvPurchase.store.manageSubscriptions();
+            isAndroid()
+              ? window.CdvPurchase.store.manageSubscriptions()
+              : onCancelSubscription(ownedProduct);
           }}
           style={{ marginLeft: '1em' }}
         >
-          <FormattedMessage {...messages.manageSubscription} />
+          {' '}
+          {isAndroid() ? (
```
Comment (RodriSanchez1) [inline src/components/Settings/Subscribe/SubscriptionInfo.js:145]: We need to add a platform validation. If the purchase was made from Paypal, this button does not have effect on Android  and viceversa. A message can be added to, something like _You made the purchase via PayPal, please manage it from the web_

File: src/components/Settings/Subscribe/SubscriptionPlans.js [hunk 4]
```diff
@@ -194,17 +227,46 @@ const SubscriptionPlans = ({
...
+                      color="primary"
+                      {...(!isLogged
+                        ? { component: Link, to: '/login-signup' }
+                        : { onClick: onSubscribe(product) })}
+                      disabled={!canPurchase}
+                    >
+                      <FormattedMessage {...messages.subscribe} />
+                    </Button>
+                  )}
+                  {!isAndroid() && (
```
Comment (RodriSanchez1) [inline src/components/Settings/Subscribe/SubscriptionPlans.js]: Use `!isCordova` instead

**Review by RodriSanchez1 — COMMENTED** [review]
Good paypal implementation!

File: src/components/Settings/Subscribe/SubscriptionInfo.js (hunk 3 above)
Comment (martinbedouret) [reply src/components/Settings/Subscribe/SubscriptionInfo.js:145]: this is now implemented

**Pull request merged**